mruby-mspec-support/mrblib/mruby-mspec-support.rb

```diff
@@ -27,10 +22,6 @@ def readlines(path)
       end
       a
     end
-
-    def split(path)
-      [File.dirname(path), File.basename(path)]
-    end
```

Comment:
I think this should get implemented in a mrbgem and not here then, since it does not prevent running other specs. But if you use it to validate mruby-require then of course it's a good workaround in the meantime.